src/ctree-element-screen/ctree-segment-container.html

```diff
@@ -187,13 +187,15 @@
 
       previous() {
         if (this._variationIndex <= 0) return;
-
+        var component = this.shadowRoot.querySelector('#component');
+        component.editing= false;
```

Comment:
It's a minor point, but I like to stress with developers that they should use consistent formatting.  This is missing a space before the `=`.